src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -2109,6 +2111,84 @@ private static void checkIntentForInboundAppLink(Context context) {
         }
     }
 
+    public void trackGestures(Activity parent) {
+        parent.getWindow().getDecorView().findViewById(android.R.id.content).setOnTouchListener(new View.OnTouchListener() {
+
+            private void resetGesture() {
+                mFirstToSecondFingerDifference = -1;
+                mSecondFingerTimeDown = System.currentTimeMillis();
+                mGestureSteps = 0;
+                mBetweenSteps = -1;
+            }
+
+            @Override
+            public boolean onTouch(View v, MotionEvent event) {
+                if(event.getPointerCount() > 2) {
+                    resetGesture();
+                    return false;
+                }
+
+                if(event.getActionMasked() == MotionEvent.ACTION_DOWN) {
+                    mFirstToSecondFingerDifference = System.currentTimeMillis();
+                }
+
+                if(event.getActionMasked() == MotionEvent.ACTION_POINTER_DOWN && (System.currentTimeMillis() - mFirstToSecondFingerDifference) < 100) {
+                    mSecondFingerTimeDown = System.currentTimeMillis();
+                    mFirstToSecondFingerDifference = 100;
```

Comment:
This magic number should be externalized since you rely on `100` later on several times.